### ISSUE TITLE
How do you preprocess the language data?

### ISSUE THREAD

**Issue by ch3cook-fdu** [zlccccc/3DVG-Transformer#2]
There are 562 scannet scenes paired with 36665 language annotations for training, and 141 scenes paired with 9508 sentences for validation. However, in your implementation, the size of your training set is 1483 and the size of your validation set is 363. I wonder where these numbers come from?

**Comment by zlccccc** [comment]
For the training samples, we follow the official split from ScanRefer. Please refer to their implementation for more details.
https://github.com/daveredrum/ScanRefer/blob/master/data/scannet/meta_data/

**Comment by ch3cook-fdu** [comment]
I mean how you organize the language annotations and scenes and why. In ScanRefer, the training set has size 36665, which differs from your 1483.🤔

**Comment by zlccccc** [comment]
In our implementation, the size of the training set and validation set are converted to 4819 and 1253 by using the lang_num_max.
During debugging, we divided the training/validation set into 1483 and 363 and forgot to delete them previously. In this version, they should be 36665 and 9508.
It has been updated now.

https://github.com/zlccccc/3DVG-Transformer/commit/95c922eea019ace94ff585bbd5dcfc89c834f4a6
https://github.com/zlccccc/3DVG-Transformer/blob/main/scripts/ScanRefer_train.py#L278
